Makefile

```diff
@@ -672,3 +676,7 @@ nova_deploy_cleanup: ## cleans up the service instance, Does not affect the oper
 	oc kustomize ${DEPLOY_DIR} | oc delete --ignore-not-found=true -f -
 	rm -Rf ${OPERATOR_BASE_DIR}/nova-operator ${DEPLOY_DIR}
 	oc rsh mariadb-openstack mysql -u root --password=${PASSWORD} -ss -e "show databases like 'nova_%';" | xargs -I '{}' oc rsh mariadb-openstack mysql -u root --password=${PASSWORD} -ss -e "drop database {};"
+
+.PHONY: kuttl_keystone
```

Comment:
1. Please add a new section with `##@` and document this target with `##` See description at L130 about the `help` target.
2. I would call this `keystone_kuttl` instead to follow the pattern of `<operator-name>_<action>` e.g. `keystone_deploy_prep`

Reply:
Good suggestions, thanks, both points are done in the last commit.